aspnetcore/index.md

```diff
@@ -16,45 +16,52 @@ uid: index
 
 By [Daniel Roth](https://github.com/danroth27), [Rick Anderson](https://twitter.com/RickAndMSFT), and [Shaun Luttin](https://twitter.com/dicshaunary)
 
-ASP.NET Core is a new [open-source](https://github.com/aspnet/home) and cross-platform framework for building modern, cloud-based, Internet-connected applications, such as web apps, IoT apps and mobile backends. It was architected to provide an optimized development framework for apps that are deployed to the cloud or run on-premises. It consists of modular components with minimal overhead, so you retain flexibility while constructing your solutions. You can develop and run ASP.NET Core apps on Windows, Mac and Linux. ASP.NET Core apps can [run on .NET Core or on the .NET Framework](https://docs.microsoft.com/dotnet/articles/standard/choosing-core-framework-server).
+ASP.NET Core is a cross-platform, high-performance, [open-source](https://github.com/aspnet/home) framework for building modern, cloud-based, Internet-connected applications. With ASP.NET Core, you can:
 
-## Why use ASP.NET Core?
+* Build web apps and services, IoT apps, and mobile backends.
+* Use your favorite development tools on Windows, macOS, and Linux.
+* Deploy to the cloud or on-premises
+* Run on [.NET Core or .NET Framework](https://docs.microsoft.com/dotnet/articles/standard/choosing-core-framework-server).
 
-ASP.NET was released over 15 years ago. Millions of developers have used ASP.NET (and continue to use it) to create web apps. ASP.NET Core is a significant redesign of ASP.NET, with architectural changes that result in a leaner and modular framework.
+## Why use ASP.NET Core?
```

Comment:
This section should tell me why I should use ASP.NET Core, but currently it reads like it's trying to convince me to use ASP.NET Core *instead of ASP.NET*. Let's reword this section so that it's focused on the benefits of ASP.NET Core and not on competing with ASP.NET. Some suggested edits below.